packages/polymath-ui/doczrc.js

```diff
@@ -0,0 +1,14 @@
+import { css } from 'docz-plugin-css';
+
+export default {
+  title: 'PolymathUI',
+  description: 'This is my awesome documentation',
+  files: '**/*.{md,mdx}',
+  // TODO @grsmto: we can't include this wrapper before it imports some polymath-js dependencies that break Docz build. Need to investigate why this is happening.
```

Comment:
What error are you getting. PolymathJS right now has a big issue where it **requires** `Contract.setParams` to be called with network parameters before anything inside it is used

Reply:
```
These dependencies were not found:

* @polymathnetwork/shared/fixtures/contracts/CappedSTOFactory.json in ../polymath-js/src/contracts/CappedSTOFactory.js
* @polymathnetwork/shared/fixtures/contracts/CountTransferManagerFactory.json in ../polymath-js/src/contracts/CountTransferManage
rFactory.js
* @polymathnetwork/shared/fixtures/contracts/PercentageTransferManagerFactory.json in ../polymath-js/src/contracts/PercentageTran
sferManagerFactory.js

To install them, you can run: npm install --save @polymathnetwork/shared/fixtures/contracts/CappedSTOFactory.json @polymathnetwor
k/shared/fixtures/contracts/CountTransferManagerFactory.json @polymathnetwork/shared/fixtures/contracts/PercentageTransferManager
Factory.json
```

Reply:
Oh, I guess it could be the babel config not having moduleResolver?